tox.ini

```diff
@@ -13,33 +13,41 @@ python =
     3.11: py311
 
 [testenv:pre-commit]
+skip_install=true
 allowlist_externals=
     pre-commit
 commands=pre-commit run --all-files --show-diff-on-failure
 
 [testenv:check-licenses]
+skip_install=true
 deps=
     liccheck>=0.7.3,<0.8
-commands=
+allowlist_externals=
+    poetry
+commands_pre=
     poetry lock
     poetry install --all-extras
     poetry show
     poetry export -f requirements.txt --output /tmp/requirements.txt
+commands=
     poetry run liccheck -s license_strategy.ini -r /tmp/requirements.txt -l PARANOID
 
-[testenv]
+[testenv:py{38,39,310,311}]
```

Comment:
No longer inherit this configuration in the other test environments, which lead to unnecessary dependency installation etc.